lnwallet/channel.go

```diff
@@ -1328,12 +1345,44 @@ func (lc *LightningChannel) fetchCommitmentView(remoteChain bool,
 		numHTLCs++
 	}
 
+	// Initiate feePerKw to the last committed fee for this chain.
+	feePerKw := commitChain.tail().feePerKw
+
+	// Check if any fee updates have taken place since that last commitment.
+	if lc.channelState.IsInitiator {
```

Comment:
Maybe I'm just still not getting it, but consider this scenario:

* Alice sends update_fee=1 to Bob, then a commitment message, then update_fee=2. 
* Bob receives the fee update, the commitment, and will send a revoke_and_ack back. This revocation will lock in the fee update for Bob. 
* Alice receives the revoke_and_ack, which should lock in the fee update for her. But out of the two update_fee messages she sent, it is update_fee=1 which should be locked in, while update_fee=2
should still be pending. 

I don't see how I can achieve this with only 1 state variable.